src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { StarRate, Wishlist, Basket } from "@shared";
+import classNames from "classnames";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+interface ProductProp {
```

Comment:
Do not duplicate interfaces. Instead create it once in src/shared/types and reuse where u need.